src/services/followMainFeedFollowing.js

```diff
@@ -1,23 +1,25 @@
-const stream = require("getstream");
+const stream = require('getstream');
+const UserService = require('../services/postgres/UserService');
 
 const followMainFeedFollowing = async (userId, userIds) => {
   if (!userIds || userIds.length == 0) {
     return;
   }
 
   const cs = stream.connect(process.env.API_KEY, process.env.SECRET);
+  const userSevice = new UserService();
   const userAdmin = await userSevice.getUserAdmin(process.env.USERNAME_ADMIN);
 
   userIds.push(userAdmin.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a potential issue here where `userAdmin` might be null or undefined, which would cause an error when trying to access `userAdmin.user_id`. You should add a null check before using `userAdmin`.
```diff
- userIds.push(userAdmin.user_id);
+ if (userAdmin && userAdmin.user_id) {
+   userIds.push(userAdmin.user_id);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->